guides/common/modules/con_deploying-virt-who-configurations.adoc

```diff
@@ -0,0 +1,9 @@
+[id="deploying-virt-who-configurations_{context}"]
+= Deploying virt-who configurations
+
+After you create a virt-who configuration, you can download a script or copy the Hammer `virt-who-config` command from the {ProjectWebUI} to deploy the configuration.
+Then, you run the script or the Hammer command to install virt-who and to create local and global virt-who configuration files.
+
+For {oVirt}, {EL} Virtualization (KVM), and {OpenStack}, you deploy the configuration on the hypervisor specified in the file.
```

Comment:
Isn't `{EL} Virtualization` effectively RHV, so `{oVirt}`? Or do you refer more to something like libvirt here?